webviews/components/timeline.tsx

```diff
@@ -222,32 +222,38 @@ function AddReviewSummaryComment() {
 	const { isAuthor } = pr;
 	const comment = useRef<HTMLTextAreaElement>();
 	return (
-		<div className="comment-form">
+		<form onSubmit={(event) => {
+			event.preventDefault();
+			submit(comment.current!.value);
+		}}>
```

Comment:
This is personal preference, but I don't like the inconsistency of having the other buttons have an `onClick` but "Submit" being up here in the form's `onSubmit`. Can you move it back the way it was?